src/containers/ConceptPage/components/ConceptMetaDataArticle.jsx

```diff
@@ -0,0 +1,145 @@
+import styled from '@emotion/styled';
+
+import { Link } from 'react-router-dom';
+import { AsyncDropdown } from '../../../components/Dropdown';
+import { fetchDraft, searchDrafts } from '../../../modules/draft/draftApi';
+
+import { toEditArticle } from '../../../util/routeHelpers';
+
+import { StyledRemoveConnectionButton } from '../../../style/LearningResourceTaxonomyStyles';
+
+import { convertFieldWithFallback } from '../../../util/convertFieldWithFallback';
+
+const StyledFlexWrapper = styled.div`
+  display: flex;
+`;
+
+const ConceptMetaDataArticle = ({ locale, t, field, articleId }) => {
+  const [article, setArticle] = useState(undefined);
+  const [modalOpen, setOpen] = useState(false);
+
+  const fetchArticle = async (articleId, locale) => {
+    const article = await fetchDraft(articleId, locale);
+    const title = convertFieldWithFallback(article, 'title', '');
+    setArticle({ ...article, title });
+  };
+
+  const searchForArticles = async inp => {
+    const articles = (await searchDrafts({
+      query: inp,
+      language: locale,
+    })).results;
+    return articles ? articles.filter(article => !!article.id) : [];
+  };
+
+  const onSelect = selectedArticle => {
+    if (selectedArticle) {
+      setSelected(selectedArticle);
+      onArticleSelectClose();
+    }
+  };
+
+  const setSelected = article => {
+    setArticle(article);
+    field.onChange({
+      target: {
+        name: field.name,
+        value: article?.id || null,
+      },
+    });
+  };
+
+  const onArticleSelectClose = () => {
+    setOpen(false);
+  };
+
+  const onArticleSelectOpen = () => {
+    setOpen(true);
+  };
+
+  const removeArticle = () => {
+    setSelected(undefined);
+  };
+
+  useEffect(() => {
+    if (articleId) {
+      fetchArticle(articleId, locale);
+    }
+  }, []);
+
+  return (
+    <div>
+      <FieldHeader title={t('form.article.label')} />
+      <Modal
+        controllable
+        isOpen={modalOpen}
+        size="regular"
+        backgroundColor="white"
+        minHeight="90vh"
+        animationDuration="0">
+        {() => (
+          <Fragment>
+            <ModalHeader>
+              <ModalCloseButton
+                title={t('dialog.close')}
+                onClick={onArticleSelectClose}
+              />
+            </ModalHeader>
+            <ModalBody>
+              <AsyncDropdown
+                idField="id"
+                name="relatedArticleSearch"
+                labelField="title"
+                placeholder={t('form.content.relatedArticle.placeholder')}
+                label="label"
+                apiAction={searchForArticles}
+                onClick={e => e.stopPropagation()}
+                onChange={onSelect}
+              />
+            </ModalBody>
+          </Fragment>
+        )}
+      </Modal>
+      {article ? (
+        <Fragment>
+          <StyledFlexWrapper>
+            <Link to={toEditArticle(article.id, article.articleType, locale)}>
```

Comment:
Du kan fjeren locale som parameter her.
```suggestion
            <Link to={toEditArticle(article.id, article.articleType)}>
```